Explorer/Assets/DCL/Infrastructure/Global/Dynamic/DynamicWorldContainer.cs

```diff
@@ -335,6 +335,10 @@ static IMultiPool MultiPoolFactory() =>
 
             //TODO should be unified with LaunchMode
             bool localSceneDevelopment = !string.IsNullOrEmpty(dynamicWorldParams.LocalSceneDevelopmentRealm);
+
+            var includedFeaturesRegistry = new FeaturesRegistry(appArgs, localSceneDevelopment);
+            FeaturesRegistry.Initialize(includedFeaturesRegistry);
```

Comment:
Can we initialize in the bootstrapper after the FeatureFlags? It would make the funcitonality more simetric, and we would be using the singleton here.